source/common/http/http2/codec_impl.cc

```diff
@@ -1980,7 +1983,8 @@ Status ClientConnectionImpl::onBeginHeaders(const nghttp2_frame* frame) {
   RELEASE_ASSERT(frame->headers.cat == NGHTTP2_HCAT_RESPONSE ||
                      frame->headers.cat == NGHTTP2_HCAT_HEADERS,
                  "");
-  RETURN_IF_ERROR(trackInboundFrames(&frame->hd, frame->headers.padlen));
+  RETURN_IF_ERROR(trackInboundFrames(frame->hd.stream_id, frame->hd.length, frame->hd.type,
```

Comment:
Will this type (nghttp2_frame) be changing in a future PR? (Same question about the enum values like NGHTTP2_HEADERS)

Reply:
Yes, that's the plan. I'll tag these PRs with https://github.com/envoyproxy/envoy/issues/23596.